homeassistant/components/wmspro/config_flow.py

```diff
@@ -0,0 +1,79 @@
+"""Config flow for WMS WebControl pro API integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import aiohttp
+import voluptuous as vol
+from wmspro.webcontrol import WebControlPro
+
+from homeassistant.components import dhcp
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN, SUGGESTED_HOST
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST, description={"suggested_value": SUGGESTED_HOST}): str,
```

Comment:
Why do we add `webcontrol` as suggested host? Let's remove that

Reply:
This is the default hostname given by Warema.

Reply:
but `webcontrol` isn't a hostname?

Reply:
Yes, it is.

Reply:
Can we use the helper for this instead of having it like this? You use the helper later on, so you know which one I mean :)

Reply:
You mean using `self.add_suggested_values_to_schema` with `webcontrol` if there is no other value already provided by DHCP?